src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -383,981 +211,204 @@
+	"mmorpg.chat.can_not_go_over_maximum_level": "Нельзя превысить максимальный уровень.",
+	"mmorpg.chat.cant_go_above": "Нельзя улучшить за пределы %1$s",
+	"mmorpg.chat.cant_go_above_max_level": "Нельзя повысить уровень предмета выше максимального: %1$s",
+	"mmorpg.chat.cant_same_buff": "У вас уже есть бафф такого типа.",
+	"mmorpg.chat.caught_something": "Вы поймали что-то!",
+	"mmorpg.chat.death_exp_loss_msg": "Из-за смерти вы потеряли %1$s опыта и получили долг в размере %2$s опыта.",
+	"mmorpg.chat.destroys_output": "При помещении в Профессиональную Станцию с другими входами/материалами, все созданные рецепты будут приносить 3 раза больше опыта, но выход будет уничтожен.",
+	"mmorpg.chat.dev_tools_enabled_contact_the_author": "Инструменты разработчика включены. Если вы видите это сообщение, пожалуйста, свяжитесь с автором Mine and Slash [robertx22], он забыл их отключить!",
+	"mmorpg.chat.enchant_upgrade_rarity": "Зачарования должны применяться последовательно. Обычное зачарование должно быть применено перед необычным зачарованием.",
+	"mmorpg.chat.favor_death_msg": "Ваша благосклонность уменьшилась на %1$s очков.",
+	"mmorpg.chat.favor_down": "§cИз-за смерти ваша благосклонность упала до %1$s. Запомните это позором.",
+	"mmorpg.chat.favor_up": "§aБлагодаря вашим достижениям, ваша благосклонность возросла до %1$s!",
+	"mmorpg.chat.infuses_stats": "Насыщает характеристики пустой экипировки.",
+	"mmorpg.chat.item_non_nbt": "Это пустой предмет, без каких-либо NBT/случайных характеристик, это означает, что вы получили его через творческий режим или команду /give. Если вы хотите протестировать мод, вы можете использовать команду /mine_and_slash give для создания предметов с работающими NBT.",
+	"mmorpg.chat.max_stacks": "Максимальные стаки: %1$s",
+	"mmorpg.chat.need_empty_socket": "Требуется пустое гнездо.",
+	"mmorpg.chat.need_pearl": "Вам нужно иметь хотя бы один %1$s в вашем инвентаре для входа.",
+	"mmorpg.chat.needs_an_affix": "У предмета должен быть аффикс.",
+	"mmorpg.chat.needs_empty_or_rune": "Требуется пустое гнездо или руну внутри предмета. Если у предмета есть руна, она будет перекатана.",
+	"mmorpg.chat.new_day": "Наступил новый день! Увеличены шансы получения профессиональных материалов для следующих предметов:",
+	"mmorpg.chat.not_correct_tier_level": "Вы пытаетесь использовать предмет с другим уровневым диапазоном, чем возможно.",
+	"mmorpg.chat.not_enough_experience": "Недостаточно опыта",
+	"mmorpg.chat.not_family": "Нельзя применить к предмету этой группы слотов.",
+	"mmorpg.chat.not_gear": "Это не предмет экипировки.",
+	"mmorpg.chat.not_jewel": "Это не ювелирный предмет.",
+	"mmorpg.chat.not_owner": "§cВы не владелец этой станции.",
+	"mmorpg.chat.not_skillgem": "Это не навыковый камень.",
+	"mmorpg.chat.not_tool": "Это не инструмент, или у инструмента пока нет характеристик.",
+	"mmorpg.chat.only_one_of_type": "Может быть только одно преимущество этого типа:",
```

Добавляю новый термин: Perk=Талант
```suggestion
	"mmorpg.chat.only_one_of_type": "Может быть только один талант этого типа:",
```

---

исправлено